src/js/login/containers/Main.jsx

```diff
@@ -214,18 +216,26 @@ class Main extends React.Component {
             </Modal>
           </div>
         );
+        break;
       }
       case 'verifyPage':
-        return this.props.profile.loading ?
+        content = this.props.profile.loading ?
           (<LoadingIndicator message="Loading the application..."/>) :
           (<Verify
             shouldRedirect={this.props.shouldRedirect}
             login={this.props.login}
             profile={this.props.profile}
             handleLogin={this.handleLogin}/>);
+        break;
       default:
-        return null;
+        content = null;
     }
+
+    return (
```

Comment:
Is there a reason to do it this way now?

Reply:
I actually would preferr returning from the switch since there's no common content between all three of those to be rendered in this chunk.